Source/ConversationV1/Tests/ConversationTests.swift

```diff
@@ -0,0 +1,215 @@
+
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import XCTest
+import WatsonDeveloperCloud
+import Freddy
+
+class ConversationTests: XCTestCase {
+
+    // MARK: - Parameters and constants
+
+    // the Conversation service
+    var service:     Conversation!
+    var workspaceID: Conversation.WorkspaceID! = "car_demo_1"
+
+    // the initial node and response from Watson
+    let initialMessage = "Hello"
+    let initialResponse = "Welcome to the Watson Car Demo..."
+
+    let secondMessage = "Turn on wipers"
+    let secondResponse = "Your wipers are now on"
+    
+    // invalid parameters for negative tests
+    let invalidWorkspaceID = "9354b734-d5b2-4fd3-bee0-e38adbcab575"
```

Comment:
It might be helpful to make this a bit more explicit. I think we switched to using something like "this-id-is-invalid" in the updated tests for Dialog.